cmd/containerd-stress/main.go

```diff
@@ -52,7 +52,7 @@ func init() {
 	ct = ns.NewLabeledTimer("run", "Run time of a full container during the test", "commit")
 	execTimer = ns.NewLabeledTimer("exec", "Run time of an exec process during the test", "commit")
 	binarySizeGauge = ns.NewLabeledGauge("binary_size", "Binary size of compiled binaries", metrics.Bytes, "name")
-	errCounter = ns.NewLabeledCounter("errors", "Errors encountered running the stress tests", "err")
+	errCounter = ns.NewLabeledCounter("errors", "Errorf encountered running the stress tests", "err")
```

Comment:
no

Reply:
Thanks a lot for review! This is actually a problem when use batch replace rather than use tools. i will re-check whole changes.